extensions/positron-r/src/commands.ts

```diff
@@ -147,8 +147,10 @@ export async function registerCommands(context: vscode.ExtensionContext) {
 		vscode.commands.registerCommand('r.sourceCurrentFile', async () => {
 			try {
 				const filePath = await getEditorFilePathForCommand();
+				// In the future, we may want to shorten the path by making it
+				// relative to the current working directory.
 				if (filePath) {
-					const command = `source(${filePath})`;
+					const command = `source(${JSON.stringify(filePath)})`;
```

Comment:
Move the `JSON.stringify()` here, since it originated when `r.sourceCurrentFile` was added. This code snippet is executed via `positron.runtime.executeCode()`, which is a different (less challenging) beast than PowerShell. Quoting for `r.task.rmarkdownRender` is handled in the setup of the `vscode.ShellExecution` instance.